index.html

```diff
@@ -26,6 +26,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <div id="mobile-nav" class="invisible">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <ul>
+            <li><a class="mobile-link" href="#projects">Portoflio</a></li>
+            <li><a class="mobile-link" href="#about-me">About</a></li>
+            <li>
+              <a class="mobile" href="#contact">Contact</a>
```

Comment:
- [ ] Please always make sure that your project is running without any issues. As I noticed that for menus `Portfolio` and `About` your functionality is working fine and directed me to the right section. Whereas when I tried to click on `Contact` the `mobile-menu` overlay was still there I was not able to see the `contact`, I had to close the `menu` first, to see the contact section. It considers a bad `UX` or `user experience` which can frustrate the user. 
- Kindly fix this issue, so that your application gives good `user experience` to the website visitors or clients.